plugin/src/main/java/org/opensearch/ml/factory/MultiGetRequestBuilderFactory.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.ml.factory;
+
+import org.opensearch.action.get.MultiGetAction;
+import org.opensearch.action.get.MultiGetRequestBuilder;
+import org.opensearch.client.Client;
+
+public class MultiGetRequestBuilderFactory {
+
+    public MultiGetRequestBuilder createMultiGetRequestBuilder(Client client) {
+        return new MultiGetRequestBuilder(client, MultiGetAction.INSTANCE);
```

Comment:
We should always avoid `new` in code since the concept of `IoC`, also `new` is a blocker of UT, using factory method is a preferred approach to avoid these.